terragrunt/aws/cloudfront/cloudfront.tf

```diff
@@ -0,0 +1,105 @@
+resource "aws_cloudfront_distribution" "url_shortener_api" {
+  enabled     = true
+  aliases     = [var.domain]
+  price_class = "PriceClass_100"
```

Comment:
If you're curious, this price class controls the number of edge locations in the world this CloudFront distribution serves:
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/PriceClass.html

So the more you pay, the more likely your response will be geographically close the requestor.

Reply:
Interesting! I guess good thing that people only in Canada will be using it. Thanks for letting me know!